BGAnimations/ScreenGameplay overlay/TrackExScoreJudgments.lua

```diff
@@ -0,0 +1,118 @@
+-- In this file, we're keep track of judgment counts for calculating the EX score to display during
+-- ScreenGameplay and ScreeEvaluation.
+--
+-- Similar to PerColumnJudgmentTracking.lua, this file doesn't override or recreate the engine's
+-- judgment system in any way. It just allows transient judgment data to persist beyond ScreenGameplay.
+------------------------------------------------------------
+
+-- don't bother tracking for Casual gamemode
+if SL.Global.GameMode == "Casual" then return end
+
+local player = ...
+
+local stats = STATSMAN:GetCurStageStats():GetPlayerStageStats(player)
+local ex_counts = {
+    -- These counts are only tracked while a player hasn't failed.
+    -- This is so that the EX score stops changing once they've failed.
+    W0 = 0,
+    W1 = 0,
+    W2 = 0,
+    W3 = 0,
+    W4 = 0,
+    W5 = 0,
+    Miss = 0,
+    HitMine = 0,
+    -- Hold and rolls are tracked together here since they're covered by
+    -- the HoldNoteScore.
+    Held = 0,
+    LetGo = 0,
+
+    -- The W0 count displayed in the pane in ScreenEvaluation should
+    -- still display the total count (whether or not the player has failed).
+    -- Track that separately.
+    W0_total = 0
+}
+
+local valid_tns = {
+    W1 = true,
+    W2 = true,
+    W3 = true,
+    W4 = true,
+    W5 = true,
+    Miss = true,
+    HitMine = true
+}
+
+local valid_hns = {
+    LetGo = true,
+    Held = true
+}
+
+return Def.Actor{
+	JudgmentMessageCommand=function(self, params)
+		if params.Player ~= player then return end
+        local count_updated = false
```

Comment:
inconsistent indentation

Reply:
Oh what the. There were mixed spaces/tabs. Changed to tabs to be consistent with most of SL for now.